2025-09-12 19:54:58.329 [info] Telemetry level is off
2025-09-12 19:54:58.329 [info] Experiments are disabled, only manually opted experiments are active.
2025-09-12 19:54:58.329 [warning] Dir "/home/user/arch-copy/.pixi/envs" is not watchable (directory does not exist)
2025-09-12 19:54:58.355 [info] Starting Environment refresh
2025-09-12 19:54:58.355 [info] Searching for interpreters in posix paths locator
2025-09-12 19:54:58.359 [info] Searching for pyenv environments
2025-09-12 19:54:58.360 [info] Searching for conda environments
2025-09-12 19:54:58.360 [info] Searching for global virtual environments
2025-09-12 19:54:58.360 [info] Searching for custom virtual environments
2025-09-12 19:54:58.362 [info] pyenv is not installed
2025-09-12 19:54:58.362 [info] Finished searching for pyenv environments: 7 milliseconds
2025-09-12 19:54:58.362 [info] Finished searching for custom virtual envs: 6 milliseconds
2025-09-12 19:54:58.366 [info] > conda info --json
2025-09-12 19:54:58.371 [info] > hatch env show --json
2025-09-12 19:54:58.371 [info] cwd: .
2025-09-12 19:54:58.408 [info] > pyenv which python
2025-09-12 19:54:58.408 [info] cwd: .
2025-09-12 19:54:58.408 [info] Finished searching for global virtual envs: 52 milliseconds
2025-09-12 19:54:58.421 [info] Found: /bin/python --> /bin/python3.13
2025-09-12 19:54:58.426 [info] Found: /bin/python3 --> /bin/python3.13
2025-09-12 19:54:58.426 [info] Found: /bin/python3.13 --> /bin/python3.13
2025-09-12 19:54:58.441 [info] Found: /sbin/python --> /sbin/python3.13
2025-09-12 19:54:58.441 [info] Found: /sbin/python3 --> /sbin/python3.13
2025-09-12 19:54:58.441 [info] Found: /sbin/python3.13 --> /sbin/python3.13
2025-09-12 19:54:58.443 [info] Found: /usr/bin/python --> /usr/bin/python3.13
2025-09-12 19:54:58.443 [info] Found: /usr/bin/python3 --> /usr/bin/python3.13
2025-09-12 19:54:58.443 [info] Found: /usr/bin/python3.13 --> /usr/bin/python3.13
2025-09-12 19:54:58.458 [info] Found: /usr/sbin/python --> /usr/sbin/python3.13
2025-09-12 19:54:58.459 [info] Found: /usr/sbin/python3 --> /usr/sbin/python3.13
2025-09-12 19:54:58.459 [info] Found: /usr/sbin/python3.13 --> /usr/sbin/python3.13
2025-09-12 19:54:58.470 [info] > /bin/python -I ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/get_output_via_markers.py ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/interpreterInfo.py
2025-09-12 19:54:58.481 [info] > /sbin/python -I ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/get_output_via_markers.py ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/interpreterInfo.py
2025-09-12 19:54:58.486 [info] Finished searching for interpreters in posix paths locator: 132 milliseconds
2025-09-12 19:54:58.490 [info] Environments refresh paths discovered (event): 136 milliseconds
2025-09-12 19:54:58.490 [info] Environments refresh paths discovered: 136 milliseconds
2025-09-12 19:54:58.503 [info] > /usr/bin/python -I ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/get_output_via_markers.py ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/interpreterInfo.py
2025-09-12 19:54:58.509 [info] > /usr/sbin/python -I ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/get_output_via_markers.py ~/.vscode-oss/extensions/ms-python.python-2025.14.0-universal/python_files/interpreterInfo.py
2025-09-12 19:54:58.534 [info] Environments refresh finished (event): 180 milliseconds
2025-09-12 19:54:58.538 [info] Environment refresh took 183 milliseconds
2025-09-12 19:54:58.543 [info] Python interpreter path: /bin/python
2025-09-12 19:54:59.203 [info] Starting Jedi language server for arch-copy.
